test/fs.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const metatests = require('metatests');
+const common = require('..');
+
+metatests.test('create 2 extra empty directories', test => {
+  common.mkdirp('test/example1/example2');
```

Comment:
You should check if the directory exists and if it does - remove it. Also, it whould be removed after the test anyway.